.travis.yml

```diff
@@ -50,3 +51,5 @@ after_success:
   - ./lighthouse-core/scripts/generate-combined-coverage.sh
 after_failure:
   - grep 'No screenshots' perf.json && travis-artifacts upload --path perf-0.trace.json
+addons:
+  chrome: stable
```

Comment:
It's probably an abstraction travis-ci is making so it doesn't matter what distro you're on
https://docs.travis-ci.com/user/chrome